treerec/tskit/tables.c

```diff
@@ -6458,7 +6458,7 @@ simplifier_insert_input_roots(simplifier_t *self)
                     if (ret != 0) {
                         goto out;
                     }
-                    simplifier_map_mutations(self, input_id, x->left, x->right, x->node);
+                    simplifier_map_mutations(self, input_id, x->left, x->right, output_id);
```

Comment:
Sounds like this change will be in a tskit point-release soon; I guess we're taking it early, and will take the rest of the tskit changes that are in that point-release with a separate PR, also for SLiM 3.5, right?  And then hopefully we're done with all this tskit fallout for a while.